docs/common/registration.md

```diff
@@ -48,6 +48,24 @@ methods:
 
 For definitions for the various fields, consult the corresponding template file for the type of data you are updating.
 
+To remove or disable a resource already registered, follow the steps:
+
+1. If the resource is a Compute Element or OSG submitter, set `Active: false` within resource group's yaml file.  For example, to remove
+   the resource GLOW, edit the file `topology/University of Wisconsin/GLOW/GLOW.yaml`, setting `Active: false`:
+    
+        :::toml
```

Comment:
Does this look any different as YAML?

Reply:
Shoot, was working on another project that used toml.  Yaml kinda works, but the %RED% stuff doesn't work when I use `:::yaml`.  So I just removed the definition line entirely.